Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -343,7 +346,7 @@ namespace O3DE::ProjectManager
             pybind11::finalize_interpreter();
         }
 
-        return !PyErr_Occurred();
+        return true;
```

Comment:
So this would also crash?
```suggestion
        return Py_IsInitialized() == 0;
```

Reply:
Nice! Thanks for the suggestion, this works